bindsnet/network/__init__.py

```diff
@@ -251,7 +252,8 @@ def run(self, inpts, time, **kwargs):
 
             # Run synapse updates.
             for c in self.connections:
-                self.connections[c].update(reward=reward)
+                self.connections[c].update(reward=reward,
```

Comment:
Please describe in the docs what mask is, what is input size, like you did for reward and clamps